doc/example/conversion_reaction.ipynb

```diff
@@ -270,7 +271,7 @@
     ")\n",
     "\n",
     "result = sample.sample(\n",
-    "    problem, n_samples=10000, sampler=sampler, result=result, filename=None\n",
+    "    problem, n_samples=1000, sampler=sampler, result=result, filename=None\n",
```

Comment:
Similarly as above, is it sufficent? Don't need to answer, but just check the figures and what they look like.
Recently, when I was updating notebooks, I was choosing the lowest numbers that didn't alter the figures too much -- such that it still looks presentable. So I would just make sure it stays that way if we're reducing it further.